origin-dapp-creator-client/src/pages/Configure.js

```diff
@@ -37,6 +36,21 @@ class Configure extends React.Component {
     this.setState({ redirect: '/metamask' })
   }
 
+  componentDidMount() {
+    // Handle the case where filter by own listings is enabled but the Ethereum
+    // address is different to the currently active web3 account (which will
+    // populate the config.marketplacePublisher attribute)
+    if (
+      this.props.config.filters.listings.marketplacePublisher &&
+      this.props.config.filters.listings.marketplacePublisher !==
```

Comment:
is it worth doing comparison in canonial lower case form ?

Reply:
Done, thanks!